libnmstate/nm/ipv6.py

```diff
@@ -120,7 +118,7 @@ def create_setting(config, base_con_profile):
         _set_static(setting_ip, ip_addresses)
     else:
         setting_ip.props.method = (
-            nmclient.NM.SETTING_IP6_CONFIG_METHOD_LINK_LOCAL
+            nmclient.NM.SETTING_IP6_CONFIG_METHOD_IGNORE
```

Comment:
These have been explicitly set to LINK_LOCAL due to the inability to disable IPv6, so I think these need to stay link-local until we manage to disable IPv6 post activation/modify.

Reply:
The `link local` just require NM to wait on link, then kernel set link local address.
The `ignore` just leave kernel and user to handle it all.

For the same thing in IPv4, we set disabled instead of link local.
So I think we are good on this ignore method.